src/v2/view-builder/views/TerminalView.js

```diff
@@ -121,6 +126,8 @@ const Body = BaseForm.extend({
       messagesObjs.value.push({ message: loc('oie.return.to.original.tab', 'login')});
     } else if (this.options.appState.containsMessageWithI18nKey('tooManyRequests')) {
       description = loc('oie.tooManyRequests', 'login');
+    } else if (this.options.appState.containsMessageWithI18nKey(IDX_RETURN_LINK_OTP_ONLY)) {
```

Comment:
If we do [this](https://github.com/okta/okta-signin-widget/pull/2235/files#r734112388) then we can get rid of this ever expanding `else if` clauses